src/LayerPlanBuffer.cpp

```diff
@@ -17,16 +17,55 @@ void LayerPlanBuffer::flush()
     }
     while (!buffer.empty())
     {
-        buffer.front().writeGCode(gcode);
+        buffer.front()->writeGCode(gcode);
         if (CommandSocket::isInstantiated())
         {
             CommandSocket::getInstance()->flushGcode();
         }
         buffer.pop_front();
     }
-    
 }
 
+void LayerPlanBuffer::addConnectingTravelMove()
+{
+    auto newest_layer_it = --buffer.end();
+    auto prev_layer_it = newest_layer_it;
+    prev_layer_it--;
+    Point destination;
+    {
```

Comment:
To show that it doesn't interact with the rest of the code in that function.
Seems a bit redundant indeed.

Want me to refactor it?